notebooks/01_data_on_manifolds.ipynb

```diff
@@ -0,0 +1,669 @@
+{
```

Comment:
<p>Why does an array of shape (2, 3) get printed? Is data still in spherical coordinates at this point or already converted to extrinsic? Maybe print <code>data[:5]</code> instead to get the coordinates that correspond to 5 cities?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/786/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
Converted to extrinsic now, since it is the principal system of coords that we use for the sphere. Can be discussed and reverted though.